nginx.conf

```diff
@@ -469,5 +469,69 @@ location = /guides/manage-event-stream/ {
 }
 
 location = /performance-monitoring/setup/ {
-  return 302 /performance-monitoring/getting-started/;
-}
+  return 302 /product/performance/getting-started/;
+}
+  location = /workflow/alerts-notifications/ {
+    return 302 /product/alerts-notifications/;
+  }
+  
+  location = /workflow/alerts-notifications/alerts/ {
+    return 302 /product/alerts-notifications/alerts/;
+  }
+  
+  location = /workflow/alerts-notifications/notifications/ {
+    return 302 /product/alerts-notifications/notifications/;
+  }
+  
+  location = /workflow/issue-owners/ {
+    return 302 /product/error-monitoring/issue-owners/;
+  }
+  
+  location = /workflow/integrations/ {
+    return 302 /product/integrations/;
+  }
+   
+  location = /workflow/dashboards/ {
+    return 302 /product/error-monitoring/dashboards/;
+  }
+   
+  location = /workflow/debug-files/ {
+    return 302 /data-management/debug-files/;
+  }
+  
+  location = /workflow/releases/ {
+    return 302 /product/releases/;
+  }
+  
+  location = /workflow/search/ {
+    return 302 /product/sentry-basics/search/;
+  }
+  
+  location = /enriching-error-data/environments/ {
+    return 302 /product/sentry-basics/environments/;
+  } 
+  
+  location = /enriching-error-data/error-tracing/ {
+    return 302 /product/sentry-basics/guides/;
+  }
+  
+  location = /guides/ {
+    return 302 /product/guides/;
+  }
+  
+  location = /platforms/javascript/sourcemaps {
```

Comment:
does this page actually cover the sourcemaps component well enough?

Reply:
It should. I worked with Kamil on integrating the sourcemaps content that was published in the JavaScript index file with the content that was published (and slightly different) in the sourcemaps file, creating one source of truth. I'm open to input, if you think something is missing.